vendor/pip-pop/pip-clean

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Usage:
+  pip-clean <req-file>
+
+Options:
+  -h --help     Show this screen.
+"""
+from docopt import docopt
+
+BAD_PACKAGES = ['appdirs', 'packaging', 'pyparsing', 'six', 'setuptools', 'distribute']
+
+def main():
+    args = docopt(__doc__, version='pip-grep')
+    req_file = args['<req-file>']
+
```

Comment:
Understood...  see other package names that begin with but do not end with `six` for instance https://pypi.python.org/pypi?%3Aaction=search&term=Six&submit=search